app/parser/v0_0_1/schema_parser.py

```diff
@@ -251,7 +248,7 @@ def _parse_question(self, schema, questionnaire):
             question.id = ParserUtils.get_required_string(schema, "id")
             question.title = ParserUtils.get_required_string(schema, "title")
             question.description = ParserUtils.get_required_string(schema, "description")
-            question.skip_condition = self._parse_skip_condition(ParserUtils.get_optional(schema, "skip_condition"))
+            question.skip_condition = schema.get("skip_condition")
```

Comment:
I don't think we should be changing how one property of the schema object works, this would make it inconsistent with every other property. Is it possible to revert this and add an as_dict method to `schema/skip_condition.py`?